docker-compose.yml

```diff
@@ -7,6 +7,7 @@ services:
     privileged: true
     volumes:
         - scripts:/usr/src/slapi/scripts
+        - brain:/usr/src/slapi/brain
```

Comment:
No, everything is created by docker is needed. But this creating a shared volume, nothing on the host itself. I will have to add documentation to mount the `/usr/src/slapi/brain` locally if they wish to keep persistence outside of the compose setup.